packages/react-textarea/src/stories/TextareaControlled.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Textarea } from '../Textarea';
+import { useId } from '@fluentui/react-utilities';
+import { Label } from '@fluentui/react-label';
+import type { TextareaProps } from '../Textarea';
+import { makeStyles, shorthands } from '@griffel/react';
+
+const useStyles = makeStyles({
+  container: {
+    '> div': { ...shorthands.padding('20px') },
+    '& label': { display: 'block', marginBottom: '10px' },
```

Comment:
And make container a flex container so that you don't need to make the label display block. You can also use gap instead of margin.